modules/core/src/main/java/org/apache/ignite/internal/managers/deployment/GridDeploymentCommunication.java

```diff
@@ -337,50 +337,63 @@ void sendUndeployRequest(String rsrcName, Collection<ClusterNode> rmtNodes) thro
                 GridIoPolicy.P2P_POOL);
         }
     }
-
+    
+    /**
+     * Queries whether a node is a direct or indirectly responsible for 
```

Comment:
Returns true if the given node is directly or indirectly responsible for the current {@linkplain GridDeploymentRequest request} (which is not an {@linkplain GridDeploymentRequest#isUndeploy undeploy) being processed by this thread.

@pre node != null